### PR TITLE
Adds a k'ois freezer subtype and places it in the kitchen

### PR DESCRIPTION
**Pull request by lavillastrangiato**
Title. The fridge also has some very obvious markers around it.
image

I am not sure why edits to some syntax in the INDRA and supermatter are showing up.

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 1]
```diff
@@ -29547,7 +29552,7 @@
 	input_tag = "cooling_in";
 	name = "Supermatter Monitoring Terminal";
 	output_tag = "cooling_out";
-	sensors = list("supermatter_sensor"="Supermatter Core")
+	sensors = list("supermatter_sensor"="Supermatter  Core")
```
Comment (Geevies) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:29555]:
```suggestion
	sensors = list("supermatter_sensor"="Supermatter Core")
```

File: maps/sccv_horizon/sccv_horizon-2_deck_2.dmm [hunk 2]
```diff
@@ -38366,7 +38371,7 @@
 /obj/machinery/computer/general_air_control/large_tank_control/terminal{
 	input_tag = "fusion_injector_air";
 	name = "INDRA Chamber Monitor";
-	sensors = list("fusion_sensor"="INDRA  Chamber")
+	sensors = list("fusion_sensor"="INDRA    Chamber")
```
Comment (Geevies) [inline maps/sccv_horizon/sccv_horizon-2_deck_2.dmm:38374]:
```suggestion
	sensors = list("fusion_sensor"="INDRA  Chamber")
```

File: code/game/objects/structures/crates_lockers/closets/secure/freezer.dm [hunk 3]
```diff
@@ -93,3 +93,12 @@
 	for(var/i = 0, i < 2, i++)
 		new /obj/item/reagent_containers/food/drinks/carton/soymilk(src)
 		new /obj/item/reagent_containers/food/condiment/sugar(src)
+/obj/structure/closet/secure_closet/freezer/kois
```
Comment (Geevies) [inline code/game/objects/structures/crates_lockers/closets/secure/freezer.dm:96]:
```suggestion

/obj/structure/closet/secure_closet/freezer/kois
```

File: code/game/objects/structures/crates_lockers/closets/secure/freezer.dm [hunk 4]
```diff
@@ -93,3 +93,12 @@
...
+	name = "k'ois freezer"
+	desc = "A freezer with k'ois inside. Don't eat this unless you're a Vaurca!"
+
+/obj/structure/closet/secure_closet/freezer/kois/fill()
+	..()
+	for(var/i = 0, i < 8, i++)
+		new /obj/item/reagent_containers/food/snacks/grown/kois(src)
+	..()
```
Comment (Geevies) [inline code/game/objects/structures/crates_lockers/closets/secure/freezer.dm:104]:
```suggestion
```

**Review by Geevies — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/freezer.dm (hunk 3 above)
Comment (lavillastrangiato) [reply code/game/objects/structures/crates_lockers/closets/secure/freezer.dm:97]: should be good?

**Review by Geevies — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**